polygeist/tools/cgeist/Test/Verification/no_inline.c

```diff
@@ -11,7 +11,7 @@ void foo(int A[10]) {
 // CHECK-LABEL: func @main()
 // CHECK: call @foo
 // OPT1-LABEL: func @main()
-// OPT1-NOT: call @foo
+// OPT1-CHECK: call @foo
```

Comment:
Maybe remove the second run line?

```suggestion
// OPT1: call @foo
```

Reply:
I want to keep it because we will do inlining later on once I am done with the current inlining PRs I am working on.

Reply:
ok, then just the suggested change.